### PR TITLE
noise profiles

### PR DESCRIPTION
**Pull request by rawfiner**

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -850,6 +850,21 @@
             {"skip": true, "name": "EOS 200D iso 51200", "iso": 51200, "a": [0.00249000518027299, 0.00112357559710744, 0.00181437909489014], "b": [1.72431912704092e-05, 2.76503246829995e-05, 2.03920295934623e-05]}
           ]
         },
+        {
+          "comment": "EOS 200D II (EOS 250D) contributed by scybernaut",
```
Comment (kmilos) [inline data/noiseprofiles.json:854]: Could add this is also EOS Rebel SL3? Maybe even better change to "eos 250d (aka eos 200d II, eos rebel sl3) ..."

File: data/noiseprofiles.json [hunk 2]
```diff
@@ -850,6 +850,21 @@
...
+          "model": "EOS 250D",
+          "profiles": [
+            {"name": "EOS 200D II iso 100", "iso": 100, "a": [5.22367254613872e-06, 2.38125889064614e-06, 4.62535308187296e-06], "b": [4.05699863614146e-09, 2.99296859087643e-09, 4.09607381684299e-09]},
```
Comment (kmilos) [inline data/noiseprofiles.json:857]: Change all "EOS 200D II" occurrences to "EOS 250D" please.

File: data/noiseprofiles.json (hunk 1 above)
Comment (rawfiner) [reply data/noiseprofiles.json:854]: Thanks for the review, I updated the comment accordingly

File: data/noiseprofiles.json (hunk 2 above)
Comment (rawfiner) [reply data/noiseprofiles.json:857]: done

**Comment by rawfiner** [comment]
Last push adds 2 new profiles in addition to solving the wording for Canon 250D

**Comment by TurboGit** [comment]
IINM OM-1 is still not supported by rawspeed, right?

**Comment by kmilos** [comment]
@TurboGit It was indeed added recently to rawspeed devel. Would be good to figure out w/ @LebedevRI how that gets in for dt 4.2.1...

**Comment by kmilos** [comment]
@TurboGit Good to go now!

**Comment by TurboGit** [comment]
Sure, and we must not forget to merge this in 4.2.1 when rawspeed branch is updated there too.

**Pull request merged**